macros/src/actix/mod.rs

```diff
@@ -169,6 +173,36 @@ pub fn emit_v2_definition(attrs: TokenStream, input: TokenStream) -> TokenStream
     gen.into()
 }
 
+fn get_field_type(field: &Field, mut is_required: Option<bool>) {
+    match field.ty {
+        Type::Path(ref p) => {
+            let ty = p
+                .path
+                .segments
+                .last()
+                .expect("expected type for struct field");
+
+            if p.path.segments.len() == 1 && &ty.ident == "Option" {
+                is_required = Some(false);
+            }
+
+            address_type_for_fn_call(&field.ty)
+        }
+        Type::Reference(_) => address_type_for_fn_call(&field.ty),
+        _ => address_type_for_fn_call(&field.ty),
+    };
+}
+
+/// Generates code for a tuple struct with fields.
+fn handle_unnamed_field_struct(fields: &FieldsUnnamed, props_gen: &mut proc_macro2::TokenStream) {
+    let field = fields.unnamed.iter().nth(0).unwrap();
+    let ty_ref = get_field_type(field, None);
+
+    props_gen.extend(quote!({
```

Comment:
Getting a, not sure why maybe coz of above 🤔 
```
error[E0277]: the trait bound `(): quote::to_tokens::ToTokens` is not satisfied
   --> macros/src/actix/mod.rs:201:22
    |
201 |       props_gen.extend(quote!({
    |  ______________________^
202 | |         schema = #ty_ref::raw_schema();
203 | |     }));
    | |______^ the trait `quote::to_tokens::ToTokens` is not implemented for `()`
    |
    = note: required by `quote::to_tokens::ToTokens::to_tokens`
    = note: this error originates in a macro outside of the current crate (in Nightly builds, run with -Z external-macro-backtrace for more info)
```

Reply:
We do this only when we get a type. If we don't get a type out of that function, we'll skip extending `props_gen` entirely.